spec/rails_helper.rb

```diff
@@ -160,6 +163,10 @@ def sign_in(resource, _deprecated = nil, scope: nil)
       allow_any_instance_of(Paperclip::MediaTypeSpoofDetector).to receive(:spoofed?).and_return(false)
       # rubocop:enable RSpec/AnyInstance
     end
+    
+    unless example.metadata[:paperclip_processing]
+      allow_any_instance_of(Paperclip::Attachment).to receive(:post_process).and_return(true) # rubocop:disable RSpec/AnyInstance
+    end
```

Comment:
This is already covered by an `attachment_processing` tag defined just a few lines before this!